js/web/lib/wasm/jsep/init.ts

```diff
@@ -45,6 +45,14 @@ class TensorViewImpl implements TensorView {
     return elementCount === 0 ? new Int32Array() : new Int32Array(this.module.HEAP8.buffer, this.data, elementCount);
   }
 
+  getUint16Array(): Uint16Array {
+    if (this.dataType !== DataType.float16) {
```

Comment:
this.dataType !== DataType.float16 && this.dataType !== DataType.uint16

Reply:
Done